Base/home/anon/Tests/run-tests-and-shutdown.sh

```diff
@@ -4,19 +4,16 @@
 echo
```

Comment:
fun fact, even though this has
```
#!/bin/Shell
# shellcheck disable=all
```
shellcheck 0.8.0 still complains about not knowing what /bin/Shell is